tasks/functions/functions.js

```diff
@@ -51,9 +69,14 @@ function getGoogle(n) {
  *    age: 42
  * }
  */
-function getUser(firstName, lastName, age) {
-    
-}
+function getUser(firstName = null, lastName = null, age = null) {
+    let object = {
+        firstName,
+        lastName,
+        age
+    };
+    return object;
```

Comment:
this also can be done in one return statement:
return {
	firstName,
        lastName,
        age
    };